src/govuk/components/character-count/template.test.js

```diff
@@ -260,4 +260,23 @@ describe('Character count', () => {
       })
     })
   })
+
+  describe('no maximum', () => {
```

Comment:
Again this could do with being clearer, maybe something like:

```suggestion
  describe('when neither maxlength nor maxwords are set', () => {
```